vali_objects/position.py

```diff
@@ -311,13 +315,18 @@ def add_order(self, order: Order):
                 f"Order trade pair [{order.trade_pair}] does not match position trade pair [{self.trade_pair}]"
             )
 
-        if self._clamp_and_validate_leverage(order):
+        if self._clamp_and_validate_leverage(order, abs(net_portfolio_leverage)):
```

Comment:
it feels like this will cause some positions to end at a leverage value lower than the minimum acceptable threshold. I think this is more likely to happen now. Can we pass on the position in such a situation, log a warning, and not raise an error?

Reply:
added an additional check while lowering leverage